### DISCUSSION TITLE
floorplan-card not updating values

### DISCUSSION THREAD

**Discussion by HaraldRuppert** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#37] [Q&A]
Hi,
I'm using the code on a standard Froeling S1-turbo with a combined buffer for heating and hot water, so I stripped away quite a bit. As stated in the documentation I've kept the ESPhome device to the default name so the entities should match yours. The dashboard, at least, works as expected. With one exception: The labels in the pictures stay at their default value. I've verified the entities in the SVG code against my HA and everything seems to match up. Already tried with different versions of the floorplan-card (not using it in any other dashboard at this time).

![image](https://github.com/user-attachments/assets/01d958f5-28cf-452a-9b8e-55ff30d1aba1)
![image](https://github.com/user-attachments/assets/9adecbf1-0ab7-4e84-b83c-12492a040101)
![image](https://github.com/user-attachments/assets/2f49a724-508c-4263-82e3-c166148b8cb9)

Does anyone have any idea?

Thanks

**Comment by HaraldRuppert** [comment]
well, it works now. Despite different browsers and cache clearing. Whatever. Happy it's all good now :-) 

Thanks for the project. You've done some amazing work here.

**Comment by GyroGearl00se** [comment]
Hi @HaraldRuppert,
sorry, couldn't get back to you earlier.

Yes, clearing the cache usually helps in HomeAssistant/Lovelace UI.
During development/testing I often opened up a browser in incognito mode (InPrivate) to avoid the need of manually clearing the cache :)

Glad it is working now for you.
